src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -23,8 +40,7 @@ Then, you can see the error in your dashboard:
 [{% asset js-index/error-message.png alt="Error in Unresolved Issues with title This is my fake error message" %}]({% asset js-index/error-message.png @path %})
 
 ## Capturing Errors
-### Capturing Errors / Exceptions {#capturing-errors}
```

Comment:
oh one thing @EvanPurkhiser is this gonna fuck shit up?

Reply:
j/k Evan. It's handled. ;)